11ty/definitions/-misia.md

```diff
@@ -0,0 +1,44 @@
+---
+title: -misia
+slug: -misia
+speech: noun
+defined: true
+sub_terms:
+  - text: Fat
+    full_title: fatmisia
+  - text: Islam
+    full_title: fatmisia
+  - text: Trans
+  full_title: transmisia
+reading:
+  - text: Beatley Library Anti-Oppression Guide
+    href: https://simmons.libguides.com/anti-oppression#s-lib-ctab-10174165-1
+  - text: 'Anti-Oppression: Anti-Fatmisia'
+    href: https://simmons.libguides.com/anti-oppression/anti-fatmisia
+---
+---
+from Greek for hate or hatred
+
+## Use
+
+-Misia can be appended to minoritised identifiers that are targeted for hate, such as to fat (fatmisia), trans (transmisia), or Islam (Islamomisia).
+
+## Issues
+
+-Phobia is Greek for 'fear of'.
+
+When used as a suffix in the context of Islamophobia, transphobia, or fatphobia, it is translated to mean that the group has a fear of individuals and communities who identify in those ways. However, the implication and subtext typically is that there is prejudice and discrimination against those groups. It falsely masks hate as fear.
+
+Additionally, phobias are one real component of anxiety disorders and mental illness, so conflating prejudice and discrimination (things that can be helped) with anxiety disorders (things that cannot necessarily be helped) additionally harms people who experience phobias from their anxiety disorders.
+
+As such, it removes the responsibility from those who exhibit prejudice and discrimination as it implies it is outside of their control.
```

Comment:
Suggest using the word phobia again here, instead of “it”, to reinforce what’s being discussed. Something like: “As such, using the term phobia...”. 

Could also remove the word “the” before responsibility. 

And then here: “implies it”, could be more specific here and say “those attitudes and behaviours” instead of ”it”.